game_board/avl/avl.py

```diff
@@ -137,7 +137,6 @@ def isIn(self, root, key):
 		Keyword arguments:
 		root -- current node being compared
 		key  -- key value being added
-		nid  -- node id
```

Comment:
Argument list was written incorrectly. Argument nid was never passed to function so I removed it from comment